### DISCUSSION TITLE
Failed to make API service using postman collection and environment using commandline

### DISCUSSION THREAD

**Discussion by nenofetch** [nylo-core/nylo#114] [Q&A]
Hi there, i'm making an API service using postman collection and environment on Nylo using command
`dart run nylo_framework:main make:api_service hepa_api --postman`
and i'm getting response error like this
![Screenshot 2024-03-11 213512](https://github.com/nylo-core/nylo/assets/22952560/a6b86a05-b7c4-4b05-b8da-7752c400fac2)
for the postman file, i following the instruction on the docs for placing each file
![image](https://github.com/nylo-core/nylo/assets/22952560/774367e9-6b1b-4234-a1b8-0c6432813248)

For my `flutter doctor` in any case if needed
![image](https://github.com/nylo-core/nylo/assets/22952560/eb07c857-3448-4a53-8631-d8e8322e1046)

is something wrong with my command or etc?
thank you for helping me 🙏

**Comment by agordn52** [comment]
Hi @nenofetch,

Thanks for sharing this.

Can you provide me with the files in your `public/assets/postman/collections` directory?
I want to see the structure, thanks 👍

**Reply by nenofetch** [reply]
hi too @agordn52 
here for the file structure of postman collections
![image](https://github.com/nylo-core/nylo/assets/22952560/b6e82d6e-cf0b-4d57-9840-ef6f02a4a2c0)

btw, i was testing it on Linux and fully work than Windows

**Reply by agordn52** [reply]
Hi @nenofetch,

Yeah, everything looks fine in your directory, I can't see anything wrong.
There must be a difference in how PowerShell and Linux handle things... Sadly, I don't have a Windows machine to test this either.

I'll advise in the docs to use Linux when running commands.
If anyone has a solution, feel free to submit a PR or start a discussion here :)
